src/apps/llamapay/llamapay.definition.ts

```diff
@@ -0,0 +1,43 @@
+import { Register } from '~app-toolkit/decorators';
+import { appDefinition, AppDefinition } from '~app/app.definition';
+import { AppAction, AppTag, GroupType } from '~app/app.interface';
+import { Network } from '~types/network.interface';
+
+export const LLAMAPAY_DEFINITION = appDefinition({
+  id: 'llamapay',
+  name: 'Llamapay',
+  description: 'LlamaPay integration for Zapper',
+  url: 'https://llamapay.io/',
+
+  groups: {
+    stream: {
+      id: 'stream',
+      type: GroupType.POSITION,
+      label: 'Stream',
+    },
+  },
+
+  tags: [AppTag.PAYMENTS],
+  keywords: [],
+  links: {},
+
+  supportedNetworks: {
+    [Network.ETHEREUM_MAINNET]: [AppAction.VIEW],
+    [Network.POLYGON_MAINNET]: [AppAction.VIEW],
+    [Network.OPTIMISM_MAINNET]: [AppAction.VIEW],
+    [Network.FANTOM_OPERA_MAINNET]: [AppAction.VIEW],
+    [Network.AVALANCHE_MAINNET]: [AppAction.VIEW],
+    [Network.ARBITRUM_MAINNET]: [AppAction.VIEW],
```

Comment:
Can you remove everything except for Ethereum here?

Reply:
I had been a little optimistic in my first run through 😆